### PR TITLE
Add target typed new examples to .Net 5 project

### PR DESCRIPTION
**Pull request by dmitry-golovinov-sonarsource**

### CODE REVIEW

File: analyzers/its/sources/Net5/Net5/S5122.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNetCore.Cors.Infrastructure;
+
+namespace Net5
```
Comment (pavel-mikula-sonarsource) [inline analyzers/its/sources/Net5/Net5/S5122.cs:3]: This doesn't have JSON (yet)

File: analyzers/its/sources/Net5/Net5/S2077.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Data.Sqlite;
+
+namespace Net5
+{
+    public class S2077
```
Comment (pavel-mikula-sonarsource) [inline analyzers/its/sources/Net5/Net5/S2077.cs:5]: This one doesn't have JSON yet

Comment (pavel-mikula-sonarsource) [inline analyzers/its/sources/Net5/Net5/S2077.cs:5]: This one is missing in the MMF and the sprint sheet

File: analyzers/its/sources/Net5/Net5/S4830.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Net5
+{
+    public class S4830
```
Comment (pavel-mikula-sonarsource) [inline analyzers/its/sources/Net5/Net5/S4830.cs:3]: This one doesn't have json (yet)

File: analyzers/its/sources/Net5/Net5/S3928.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Net5
+{
+    public class S3928
```
Comment (pavel-mikula-sonarsource) [inline analyzers/its/sources/Net5/Net5/S3928.cs:5]: This one doesn't have a JSON (yet)

File: analyzers/its/sources/Net5/Net5/S4433.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿using System.DirectoryServices;
+
+namespace Net5
+{
+    public class S4433
```
Comment (pavel-mikula-sonarsource) [inline analyzers/its/sources/Net5/Net5/S4433.cs:5]: This one doesn't have json (yet)

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Looks good overall, needs to wait for other PRs

File: analyzers/its/sources/Net5/Net5/S4433.cs (hunk 5 above)
Comment (dmitry-golovinov-sonarsource) [reply analyzers/its/sources/Net5/Net5/S4433.cs:5]: It was in JSON. May be you've wanted to add something else?

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**